install/kubernetes/cilium/templates/cilium-agent/servicemonitor.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   namespace: {{ .Values.prometheus.serviceMonitor.namespace | default .Release.Namespace }}
   labels:
     app.kubernetes.io/part-of: cilium
+    release: prometheus
```

Comment:
is this label a must ? or alternatively can it be configured via .Values.prometheus.serviceMonitor.labels?

Reply:
Hi @sayboras, 

I have found that there is a similiar related issue in prometheus-community. https://github.com/prometheus-community/helm-charts/issues/1911. It seems adding label is not a must, the issue can be solved by adding prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues: false.

Thanks for your reply.

Reply:
Seems like we can close this PR, feel free to let us know if you have any question/concern in Cilium slack.

Looking forward to your next contribution, thanks.